### DISCUSSION TITLE
Training not running on GPU

### DISCUSSION THREAD

**Discussion by evanviera** [acids-ircam/RAVE#252] [General]
Hey guys, I've successfully installed Rave, preprocessed my dataset, and got the training running. However, it doesn't appear to be using my GPU (looking at Nvidia-smi) , which is an NVIDIA GeForce RTX 3060 Ti. I also installed rave within a Venv environment. Has anyone had this issue? I'm a bit stuck and am looking for some direction.

RAVE
`
train set: 708 examples
val set: 15 examples
selected gpu: []
GPU available: False, used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
`

NVIDIA-SMI
`
NVIDIA GeForce RTX 3060 Ti WDDM|
NVIDIA-SMI 531.41        
Driver Version: 531.41       
CUDA Version: 12.1
`

**Comment by martinocassanelli** [comment]
same problem here on windows 10
NVIDIA GeForce RTX 3070

**Comment by julienrobert** [comment]
same problem here on windows 10 with RTX4090
